drivers/azure/util.go

```diff
@@ -96,6 +97,11 @@ func (d *Driver) getSecurityRules(extraPorts []string) (*[]network.SecurityRule,
 		mkRule(300, "DockerAllowAny", "Allow docker engine access (TLS-protected)", "*", fmt.Sprintf("%d", d.DockerPort)),
 	}
 
+	if d.OS == drivers.WINDOWS {
```

Comment:
we shouldn't be enabling SSH port for windows, so this should be

    if linux {add ssh rule } elseif windows {add winrm rule}